pkgs/by-name/db/dbvisualizer/package.nix

```diff
@@ -0,0 +1,60 @@
+{
+  lib,
+  makeDesktopItem,
+  makeWrapper,
+  stdenv,
+  temurin-jre-bin-17,
```

Comment:
That does not solve the problem. `fetchTarball` is not a package from nixpkgs but a built-in function. Adding `fetchTarball` just makes Nix build error out with the following:

```log
error: evaluation aborted with the following error message: 'lib.customisation.callPackageWith: Function called without required argument "fetchTarball" at /home/boldi/src/boldi-nixpkgs/pkgs/by-name/db/dbvisualizer/package.nix:7'
```

This makes perfect sense, since it is not the name of a package.